SPECS/glibc/glibc.spec

```diff
@@ -44,6 +44,13 @@ Patch18:        CVE-2021-33574-0001.patch
 Patch19:        CVE-2021-33574-0002.patch
 # CVE-2021-38604 is as issues introduced with the original CVE-2021-33574 CVE. 
```

Comment:
Linter found an empty space at the end of this line. Can you please remove.

Reply:
Removed extra space.